Archivos necesarios

charo007 opened the pull request:
Agregué estos archivos que son necesarios para el funcionamiento
de la página.

<!-- Saludos, si usted esta leyendo esto, significa que quiere colaborar con el proyecto aportando quizás mejoras o información que encontró mientras utilizo el mismo, es por eso que queremos agradecerle que se tome su tiempo por hacer lo antes mencionado. -->

<!-- 
	Utilice la siguiente forma de títulos-
•	Si es una mejora en la base de datos [DB] titulo
•	Si es una mejora de estilos [CSS] titulo
•	Si es una mejora en el código [HTML] titulo
Si utiliza varias mejoras, no especifique nada, solo el titulo.

-->JS y Imágenes

<!-- Necesitamos el que titulo del tema sea lo mas representativo posible. -->

##### ¿QUÉ CAMBIO PROPONE?:
<!--
	Agregue un pequeño texto que explique cual es el bug que desea reportar o cual es la situación de mejora que le gustaría que apliquemos. 
-->

Estos archivos son necesarios para el funcionamiento de la página. Los archivos .js tenía que agregarlos según la herramienta que utilicé (Boostrap) y las imágenes son las que utilizo en el carrusel.


##### NOTAS EXTRA:
<!--
Cualquier información que pueda ayudar a los desarrolladores a identificar y solucionar el problema debe ser puesta aquí.
-->

##### BRANCH(ES):
<!-- 
Especifique la(s) rama(s) afectada(s) por este problema: master, 0.x, 1.x, u otra rama. 
Reemplace el valor de master por la rama correspondiente.
-->

master



##### OTRAS PERSONALIZACIONES:
<!-- 
En caso de que se produzca alguna mejora en el código, avísennos, caso contrario, no es necesario que usted escriba nada aquí. (solo para programadores)
-->




<!-- ------------------------- THE END ------------------------------
 Gracias nuevamente por su colaboración, atenderemos su tema lo antes posible. 
-->

charo007 committed:
Archivos necesarios

Agregué estos archivos que son necesarios para el funcionamiento
de la página.

RainTrap reviewed (changes requested):
Dentro del los archivos bootstrap.bundle.min.js y jquery.slim.min.js, al final de los mismos, se solicita que se le agregue un espacio para evitar problemas.

charo007 committed:
Correción de los archivos JS

Realicé la modificación del pedido que me solicitó el administrador.
Además, agregué el espacio final en otro archivo .js en el cual
faltaba el mismo.

RainTrap reviewed (approved):
Gracias por realizar los cambios solicitados. El código esta aprobado.
